lib/database/adapters/RunAdapter.js

```diff
@@ -11,6 +11,17 @@
  * or submit itself to any jurisdiction.
  */
 
+const LHC_REVOLUTION = 11245; // The LHC revolution frequency: 11.245kHz (~ c/26.7km).
+const COLLIDING_LHC_BUNCH_CROSSINGS_IN_FILL_SCHEMA_REGEX = /[A-Za-z0-9]+_[A-Za-z0-9]+_[0-9]+_([0-9]+)_.*/;
```

Comment:
you can define it as normal variable in the following function

Reply:
applied